i18n/po/i18n_nl.po

```diff
@@ -1039,19 +1038,19 @@ msgid "child CTerminology code contains more than one constraint, that is not va
 msgstr ""
 
 #: ../aom/src/main/java/com/nedap/archie/aom/primitives/CTerminologyCode.java:280
-#, fuzzy, java-format
-msgid "child terminology constraint value code {0} does not conform to parent constraint with value code {0}"
-msgstr "De cardinaliteit {0} van dit attribuut komt niet overeen met de cardinaliteit van het attribuut dat gespecializeerd wordt: {1}"
+#, java-format
+msgid "child terminology constraint value code {0} does not conform to parent constraint with value code {1}"
+msgstr "code {0} in terminologie van specialisatie komt niet overeen met dat van de parent met code {1}"
```

Comment:
This translation does not look right to me. Perhaps just re-use some English words in the Dutch one, like 'constraint'.

```suggestion
msgstr "child terminology constraint code {0} komt niet overeen met dat van de parent constraint met code {1}"
```